checks.go

```diff
@@ -34,6 +34,7 @@ type Check struct {
 	StringMatch       string            `json:"string_match,omitempty"`
 	MuteUntil         string            `json:"mute_until,omitempty"`
 	DisabledLocations []string          `json:"disabled_locations,omitempty"`
+	Recipients        []string          `json:"recipients,omitempty"`
```

Comment:
to make it clearer to the end user, I reckon we should use `RecipientIDs` as a name